rc/filetype/kotlin.kak

```diff
@@ -0,0 +1,334 @@
+    SQLInvalidAuthorizationSpecException SQLNonTransientConnectionException
+    SQLNonTransientException SQLRecoverableException SQLSyntaxErrorException SQLTimeoutException
+    SQLTransactionRollbackException SQLTransientConnectionException SQLTransientException
+    SQLWarning SSLException SSLHandshakeException SSLKeyException SSLPeerUnverifiedException
+    SSLProtocolException StoppedException StreamCorruptedException StringConcatException
+    StringIndexOutOfBoundsException StubNotFoundException SyncFactoryException
+    SyncFailedException SyncProviderException TimeLimitExceededException TimeoutException
+    TooManyListenersException TransformerConfigurationException TransformerException
+    TransformException TransportTimeoutException TypeNotPresentException UncheckedIOException
+    UndeclaredThrowableException UnexpectedException UnknownAnnotationValueException
+    UnknownDirectiveException UnknownElementException UnknownEntityException
+    UnknownFormatConversionException UnknownFormatFlagsException UnknownGroupException
+    UnknownHostException UnknownHostException UnknownObjectException UnknownServiceException
+    UnknownTypeException UnmappableCharacterException UnmarshalException
+    UnmodifiableClassException UnmodifiableModuleException UnmodifiableSetException
+    UnrecoverableEntryException UnrecoverableKeyException UnresolvedAddressException
+    UnresolvedReferenceException UnsupportedAddressTypeException UnsupportedAudioFileException
+    UnsupportedCallbackException UnsupportedCharsetException UnsupportedEncodingException
+    UnsupportedFlavorException UnsupportedLookAndFeelException UnsupportedOperationException
+    UnsupportedTemporalTypeException URIReferenceException URISyntaxException UserException
+    UserPrincipalNotFoundException UTFDataFormatException VMCannotBeModifiedException
+    VMDisconnectedException VMMismatchException VMOutOfMemoryException VMStartException
+    WebSocketHandshakeException WriteAbortedException WritePendingException
+    WrongMethodTypeException XAException XMLParseException XMLSignatureException
+    XMLStreamException XPathException XPathException XPathExpressionException
+    XPathFactoryConfigurationException XPathFunctionException ZipException ZoneRulesException'
+
+  # ------------------------------------------------------------------------------------------------ #
+
+  join() { sep=$2; eval set -- $1; IFS="$sep"; echo "$*"; }
+
+  # ------------------------------------------------------------------------------------------------ #
+
+  printf %s\\n "declare-option str-list main_static_words $(join "${kotlin_types}" ' ')"
+
+  printf %s\\n "declare-option str-list docs_static_words $(join "${kdocs}" ' ')"
+
+  printf %s\\n "declare-option str-list error_static_words $(join "${kotlin_errors_ALL} ${java_errors_ALL}" ' ')"
+
+  printf %s\\n "declare-option str-list exception_static_words $(join "${kotlin_exceptions_ALL} ${java_exceptions_ALL}" ' ')"
```

Comment:
Those options should probably be hidden, and be kotlin_ prefixed, and I am not sure they need to be split in 4, as we always add them together. We could just have a single `kotlin_static_words` option.